File: src/main/java/wraith/fabricaeexnihilo/api/recipes/barrel/MilkingRecipe.java
package wraith.fabricaeexnihilo.api.recipes.barrel;

import alexiil.mc.lib.attributes.fluid.volume.FluidVolume;
import wraith.fabricaeexnihilo.api.crafting.EntityTypeIngredient;

public record MilkingRecipe(EntityTypeIngredient entity, FluidVolume result, int cooldown) {

    public EntityTypeIngredient getEntity() {
        return entity;
    }

    public FluidVolume getResult() {
        return result;
    }

    public int getCooldown() {
        return cooldown;
    }

}

File: src/main/kotlin/exnihilofabrico/api/recipes/barrel/MilkingRecipe.kt
package exnihilofabrico.api.recipes.barrel

import alexiil.mc.lib.attributes.fluid.volume.FluidVolume
import exnihilofabrico.api.crafting.EntityTypeIngredient

data class MilkingRecipe(val entity: EntityTypeIngredient, val result: FluidVolume, val cooldown: Int)